sdk/nodejs/output.ts

```diff
@@ -1022,3 +1022,12 @@ export function jsonStringify(obj: Input<any>, replacer?: (this: any, key: strin
         return JSON.stringify(o, replacer, space);
     });
 }
+
+/**
+ * [jsonParse] Uses JSON.parse to deserialize the given Input JSON string into a value.
+ */
+export function jsonParse(text: Input<string>, reviver?: ((this: any, key: string, value: any) => any | (number | string)[]) | undefined): Output<any> {
```

Comment:
Use the same signature for `reviver` as in the [TypeScript type definition](https://github.com/microsoft/TypeScript/blob/68df4befd68c0325861bd5b87c6e4d251ffd5a41/src/lib/es5.d.ts#L1119)?

```typescript
parse(text: string, reviver?: (this: any, key: string, value: any) => any): any;
```

Reply:
Huh managed to mangle the copy paste of that somehow first time, fixed.